backend/app/services/tool_service.py

```
@@ -0,0 +1,760 @@
+        db.add(tool)
+        db.commit()
+        db.refresh(tool)
+
+        logger.info(f"Created tool: name={tool.name}, user_id={user_id}")
+        return tool
+
+    def get_tool(self, db: Session, tool_id: int) -> Optional[Tool]:
+        """Get tool by ID"""
+        return (
+            db.query(Tool)
+            .filter(Tool.id == tool_id, Tool.is_active == True)
```

Ruff (0.14.10)

Avoid equality comparisons to `True`; use `Tool.is_active:` for truth checks

Replace with `Tool.is_active`

(E712)